stripe/src/main/java/com/stripe/android/paymentsheet/model/PaymentSheetViewState.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.paymentsheet.model
+
+/**
+ * This will show the state of the [PaymentSheetActivity] as it does work.  The states always
+ * progress as follows: Ready -> StartProcessing -> FinishProcessing -> ProcessResult
+ */
+internal sealed class PaymentSheetViewState {
```

Comment:
This also has the nice side effect of not having two classes named `PaymentSheet` in our repo :)